Gemfile.lock

```diff
@@ -22,30 +22,23 @@ GEM
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
     arel (6.0.4)
-    builder (3.2.3)
-    bump (0.5.0)
-    byebug (3.4.0)
-      columnize (~> 0.8)
-      debugger-linecache (~> 1.2)
-      slop (~> 3.6)
-    columnize (0.8.9)
-    concurrent-ruby (1.1.5)
-    debugger-linecache (1.2.0)
+    builder (3.2.4)
```

Comment:
I wonder if you run bundle update with the `--conservative` mode? 🤔 
Or all of these bumps are required to make it work? or all of them are desired?